src/index.js

```diff
@@ -1 +1,41 @@
-import './style.css';
\ No newline at end of file
+import './style.css';
+
+const todoList = document.querySelector('#todoList');
+
+const todoData = [
+  {
+    description: 'Reading about api',
+    completed: true,
+    index: 0,
+  },
+  {
+    description: 'Reading about dom manipulation',
+    completed: false,
+    index: 0,
+  },
+
+  {
+    description: 'Joining morning standup',
+    completed: false,
+    index: 0,
+  },
+];
+
+const createTodo = (todolists) => {
+  const todos = document.createElement('li');
+  todos.classList.add('todos');
+  todos.innerHTML = `
+                    <input type="checkbox" id="checked" ${todolists.completed ? 'checked' : ''} >
+                    <input type="text" id="mylist" value= "${todolists.description}">
+                    <i class="fa-solid fa-ellipsis-vertical move"></i>
+                    <button id="delete"><i class="fa-solid fa-trash-can"></i></button>`;
+  todoList.appendChild(todos);
+};
+
+const addTodo = () => {
+  todoData.forEach((todo) => {
+    createTodo(todo);
+  });
```

Comment:
- [ ] Kindly sort the list based on their index value before displaying it as per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md#:~:text=The%20list%20should%20appear%20in%20order%20of%20the%20index%20values%20for%20each%20task.).